Reaqtive/Core/Tests.Reaqtive.Linq/Operators/DefaultIfEmpty.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Reaqtive;
+using Reaqtive.Testing;
+using Reaqtive.TestingFramework;
+
+namespace Test.Reaqtive.Operators
+{
+    [TestClass]
+    public partial class DefaultIfEmpty : OperatorTestBase
+    {
+        [TestInitialize]
+        public void Initialize()
+        {
+            base.TestInitialize();
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            base.TestCleanup();
+        }
+
+        [TestMethod]
+        public void DefaultIfEmpty_SaveAndReload()
+        {
+            var state = Scheduler.CreateStateContainer();
+
+            var checkpoints = new[] {
+                OnSave(290, state),
+                OnLoad(305, state),
+            };
+
+            var xs = Scheduler.CreateHotObservable(
```

Comment:
This test case is good, but we should also have a test case where we don't have `OnNext` elements and just an `OnCompleted`, with checkpoint/recovery before it:

```
OnSave(210)
OnLoad(220)
OnCompleted(230)
```

and then expect to see

```
OnNext(230, 0)
OnCompleted(230)
```

This test then makes sure that the `false` value is stored correctly as well.